readthedocs/projects/models.py

```diff
@@ -1446,6 +1446,13 @@ class Domain(TimeStampedModel, models.Model):
         blank=True,
     )
 
+    active = models.BooleanField(
```

Comment:
Is it needed to exclude this field from forms so is not accessed by users?

Reply:
Nope https://github.com/readthedocs/readthedocs.org/blob/5b466a108bdf4c8a69e7452509fc9daba95d9552/readthedocs/projects/forms.py#L618-L618